homeassistant/components/smartthings/__init__.py

```diff
@@ -35,6 +39,33 @@
     return True
 
 
+async def async_migrate_entry(hass: HomeAssistantType, entry: ConfigEntry):
+    """Handle migration of a previous version config entry.
+
+    A config entry created under a previous version must go through the
+    integration setup again so we can properly retrieve the needed data
+    elements.  Force this by removing the entry and triggering a new flow.
```

Comment:
Remove double space.

Reply:
Double spaces and the oxford comma, two things I have to adjust when working in HASS. :)